docs/pages/development/installation.md

```diff
@@ -14,37 +14,32 @@ If you're just starting your project, you can create a new project from our temp
 
 <Terminal cmd={["$ npx crna -t with-dev-client"]} />
 
-If you have an existing project, you'll need to [install the package and make a few changes](installation.md) to your **AppDelegate.m**, **MainActivity.java** and **MainApplication.java**.
+Otherwise, follow this guide to add `expo-dev-client` to your existing React Native or Bare project.
 
-If you created your project with `expo init` or `create-expo-app`, or you already have `expo` and/or other Expo modules up and running, use the tabs marked **With Expo modules**.
+## 0. Set up Expo modules
```

Comment:
```suggestion
## 1. Set up Expo modules
```

As per our [writing style guide here](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md), for consistency reasons, let's start numbered lists with `1` 😄